defrag/__main__.py

```diff
@@ -38,9 +41,30 @@ def main() -> None:
                 "Can't have two modules with the same name! Please change one")
 
 
-@app.get("/")
-async def root() -> Dict[str, str]:
-    return {"message": "Hello World"}
+@app.on_event("startup")
+async def startup() -> None:
+    with RedisPool() as conn:
+        conn.flushall()
+    reddit_strategy = CacheStrategy(RedisCacheStrategy(
```

Comment:
Can we somehow do this differently, so that it is called in the module file? This kinda breaks the modular aspect of defrag.

Reply:
Sure, this block is the main reason why this is a Draft PR.